CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorTopComponent.java

```diff
@@ -123,21 +120,20 @@ public AttributeEditorTopComponent() {
         setToolTipText(Bundle.HINT_AttributeEditorTopComponent());
 
         refreshRunnable = () -> {
-            try {
-                final ArrayList<Object> devNull = new ArrayList<>();
-                while (queue.drainTo(devNull) > 0) {
-                    Thread.sleep(50);
-                }
 
-                if (reader != null) {
-                    attributePanel.updateEditorPanel(reader.refreshAttributes());
-                }
-            } catch (final InterruptedException ex) {
-                LOGGER.log(Level.SEVERE, "Thread was interrupted");
-                Thread.currentThread().interrupt();
+            final ArrayList<Object> devNull = new ArrayList<>();

```

Comment:
noting it appears you've just moved this out of the try block but are you possibly able to change this declaration to `List` rather than `ArrayList` (instantiation is fine)?